### PR TITLE
Fixes various typos in the documentation

### LINKED ISSUE #302: [docs] replace phrase: "for completion" -> "for completeness"
In [these cases](https://github.com/search?q=repo%3ANVIDIA-Omniverse%2Forbit+%22for+completion%22&type=code) I think it's more correct to use "for completeness" instead of "for completion". Am I right?

### PR DESCRIPTION
**Pull request by VladimirFokow**
## Description

- fix typo: "For completion" with "For completeness" 
- fix typo: reset 
- fix typo: blocks 
- add my name to CONTRIBUTORS.md
- fix typo: remove "is"

Fixes #302, #303, #304

This PR is a squash of these previous pull requests: 
#310, #311, #312, #315

## Type of change

- This change requires a documentation update

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./orbit.sh --format`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run all the tests with `./orbit.sh --test` and they pass
- [ ] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

File: docs/source/setup/sample.rst [hunk 1]
```diff
@@ -228,3 +230,16 @@ To view the logs, run:
...
+
+   In some examples above the ``--checkpoint`` argument is used.
+   It is not required: if not specified, the latest ``model.pt`` is taken automatically.
+
+   Example usage: ``--checkpoint model_999.pt``
+
+   You can also specify the run where to take the checkpoint from, for example: 
+   ``--load_run 2024-03-11_16-11-38``
+
+   The model checkpoints are saved and loaded from the ``logs`` directory (see above).
```
Comment (Mayankm96) [inline docs/source/setup/sample.rst]: This is pretty vague IMO. We only have load_run for RSL-RL case. For others this is misleading/unclear

File: docs/source/setup/sample.rst [hunk 2]
```diff
@@ -177,7 +177,7 @@ from the environments into the respective libraries function argument and return
       # note: we enable cpu flag since SB3 doesn't optimize for GPU anyway
       ./orbit.sh -p source/standalone/workflows/sb3/train.py --task Isaac-Cartpole-v0 --headless --cpu
       # run script for playing with 32 environments
-      ./orbit.sh -p source/standalone/workflows/sb3/play.py --task Isaac-Cartpole-v0 --num_envs 32 --checkpoint /PATH/TO/model.zip
+      ./orbit.sh -p source/standalone/workflows/sb3/play.py --task Isaac-Cartpole-v0 --num_envs 32 --checkpoint model_filename.pt
```
Comment (Mayankm96) [inline docs/source/setup/sample.rst]: Did you check that for all of these it make sense? I don't think we handle all checkpoint loading the same way.

Comment (Mayankm96) [reply docs/source/setup/sample.rst]: Also this is a zip file. Please make sure if it pth/pt/zip for every framework. They have their own style.

**Review by Mayankm96 — CHANGES_REQUESTED** [review]
The checkpoint loading still seems not the clearest to me after your changes.

**Comment by VladimirFokow** [comment]
My knowledge stops at this point.
Unfortunately, I don't have the ability to figure this out in the near future.

**Comment by Mayankm96** [comment]
No worries. If you remove the checkpoint loading changes, we can look into clarifying that part from our side. Thanks!

**Pull request merged**